.rubocop.yml

```diff
@@ -11,6 +11,9 @@ inherit_from:
 Rails:
   Enabled: true
 
+Rails/ContentTag:
```

Comment:
What's the reason you disable https://docs.rubocop.org/rubocop-rails/cops_rails.html#railscontenttag here? Too many hits? If so, shouldn't it be in the TODO file?

Reply:
The commit message says:

The cop is buggy before rubocop-rails 2.12.0 [1][2], but to get that we'd need a newer theforeman-rubocop, which generates more offenses than I am willing to fix today. Disable the cop like it was disabled in foreman_ansible too [3].

[1] https://github.com/rubocop/rubocop-rails/issues/260
[2] https://github.com/rubocop/rubocop-rails/pull/526
[3] https://github.com/theforeman/foreman_ansible/pull/357

Reply:
Would it be a good idea to mention that in the file itself so future maintainers don't need to start the same archeological search?

Reply:
Can do that, sure.